docker/venice-client/sample-data/multi-dc-configs/batch-push-job.properties

```diff
@@ -0,0 +1,15 @@
+venice.urls=http://venice-controller.dc-parent.venicedb.io:5555
```

Comment:
This is fully removed. Only the "discover" one is used now.

Reply:
Ack